site/styles/_meta-detail.scss

```diff
@@ -221,15 +229,20 @@ s72-element-switcher,
   }
 
   .meta-detail-synopsis {
+    margin-bottom: -4px;
     max-width: 728px;
+
+    @include media-breakpoint-up(xs) {
+      padding-bottom: 4px;
```

Comment:
Doesn't the negative margin cancel this out?

Reply:
- Changed to margin-bottom 0